pkg/persistence/account/writer/writer.go

```diff
@@ -0,0 +1,169 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package writer
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account"
+	persistence "github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/persistence/account"
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+	"path/filepath"
+)
+
+type Context struct {
+	Fs            afero.Fs
+	OutputFolder  string
+	ProjectFolder string
+}
+
+func WriteAccountResources(writerContext Context, resources account.Resources) error {
+
+	outputFolder, err := filepath.Abs(writerContext.OutputFolder)
+	if err != nil {
+		return err
+	}
+	if err := createFolderIfNoneExists(writerContext.Fs, outputFolder); err != nil {
+		return err
+	}
+	projectFolder := filepath.Join(outputFolder, writerContext.ProjectFolder)
+	if err := createFolderIfNoneExists(writerContext.Fs, projectFolder); err != nil {
+		return err
+	}
+
+	var errOccurred bool
+	if len(resources.Policies) > 0 {
+		policies := toPersistencePolicies(resources.Policies)
+		if err := persistToFile(policies, writerContext.Fs, filepath.Join(projectFolder, "policies.yaml")); err != nil {
+			errOccurred = true
+			log.Error("failed to persist policies: %w", err)
```

Comment:
```suggestion
			log.Error("Failed to persist policies: %w", err)
```

Logging should start with uppercase. Same with the loglines below

Reply:
Thanks! Left over fmt.Errorf text from refactoring an initial version that returned on the first error instead of logging & carrying on.